addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -68,6 +70,36 @@ DropzonePlugin.include({
     },
 });
 
+SnippetsEditor.Class.include({
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Toggle visibility of Mega Menu snippets when a mega menu dropdown is shown/hidden
```

Comment:
Doc:
- Conjugated verb so "Toggles"
- max 80 chars per line
- This is not a private method anymore... what about removing the private tag and updating the "private" section to "public"??